.gitignore

```diff
@@ -1,8 +1,8 @@
 .secret
 node_modules
 build
-cache
-artifacts
+cache/
```

Comment:
I am not sure we need the slash at the end.
Adding it was just an experiment but didn't seem to work.

Reply:
I agree